README.md

```diff
@@ -13,7 +13,16 @@ Bevy is a refreshingly simple data-driven game engine built in Rust. It is free
 
 ## WARNING
 
-Bevy is still in the _very_ early stages of development. APIs can and will change (now is the time to make suggestions!). Important features are missing. Documentation is sparse. Please don't build any serious projects in Bevy unless you are prepared to be broken by API changes constantly.
+Bevy is still in the early stages of development.
+APIs can and will change in both trivial and critical ways.
+Important features are incomplete. 
+Documentation can be spotty.
+
+If you need something, please feel free to [help out](CONTRIBUTING.md)!
+We're always looking for contributors, and your hands-on experience with the engine is invaluable when deciding how something should work.
+
+If you're building a project in Bevy, you will be seriously broken by API changes every release (every 3 months or so).
```

Comment:
I don't like the "you will be seriously broken" part, it reads a bit weird to me, but it was there before so 🤷‍♂️

Reply:
IME there will be more than one line migrations for complex projects every release. You need to put some thought into some of the changes.

Reply:
Oh, I agree it will be hard and plenty of things will be broken, but my brain keeps thinking the you is referencing me not the project

Reply:
My brain reads it as if it's saying that you, the reader, will be broken because of how hard it will be

Reply:
```suggestion
If you're building a project in Bevy, it will be seriously broken by API changes every release (every 3 months or so).
```

Reply:
> it was there before

Previously it said "serious projects will be broken"

It now says "[your projects] will be seriously broken"

Those are different things, right?